docs/code/ExplainabilityCookBook.md

```diff
@@ -0,0 +1,90 @@
+# Explainability CookBook
+
+This document serves to provide different ways to use the explainability features of ML.NET.
+
+All of these samples will use the [housing data](https://github.com/dotnet/machinelearning/blob/master/test/data/housing.txt) and will reference the below data schema class and pipeline.
+
+```csharp
```

Comment:
We should try to make sure that the cookbooks code compiles and runs properly even with the ongoing changes. 
The way we do this is by adding all the code and utilities in a test in https://github.com/dotnet/machinelearning/blob/master/test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamplesDynamicApi.cs

We then copy the most important parts of the CookbookSamplesDynamicApi.cs to the .md file.

If you could do that it would be great!